code/modules/clothing/head/helmet.dm

```diff
@@ -209,6 +209,21 @@
 	worn_y_dimension = 64
 	custom_price = 350
 
+/obj/item/clothing/head/helmet/boatbomber
+	name = "gnome hat"
```

Comment:
I triple dog dare you to add a special gnome sound effect for if you manage to get a 👌 on-examine bop off while wearing a gnome hat